apps/central-scan/frontend/src/components/toggle_test_mode_button.tsx

```diff
@@ -64,9 +64,19 @@ export function ToggleTestModeButton(): JSX.Element | null {
       />
       {flowState === 'confirmation' && (
         <Modal
-          title="Switch to Test Mode"
+          title={isTestMode ? 'Switch to Official Mode' : 'Switch to Test Mode'}
           content={
-            <P>Switching to test mode will clear all scanned ballot data.</P>
+            isTestMode ? (
+              <P>
+                Switching to official mode will delete all test ballot batches
+                and CVRs.
+              </P>
+            ) : (
+              <P>
+                Switching to test mode will delete all official ballot batches
+                and CVRs.
+              </P>
```

Comment:
Copy here should use "official ballot mode" and "test ballot mode"

Reply:
The toggles say "ballot mode" but the headers, buttons, and modal text just say "test mode." I thought that dropping "ballot" was a reasonable shorthand and I wanted to avoid long button titles. Do you think that it's semantically important to keep "ballot?"

Can circle back in the next one.

Reply:
I was just going for consistency with everywhere else. A while back I remember we decided to rename these modes from "Test Mode" to "Test Ballot Mode" to make it more clear what they were so we probably just missed this spot. I don't feel so strongly about this modal, but would generally lean towards consistency.